### PR TITLE
fix: download button for Metropolis font

### PR DESCRIPTION
**Pull request by astorije**
## PR Checklist

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Screen Shot 2021-01-08 at 11 13 56 PM

## What is the new behavior?

Screen Shot 2021-01-08 at 11 13 45 PM

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/website/get-started/design.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The default font intended for use with Clarity is [Clarity City](https://github.
 Clarity Angular uses Metropolis as the base font, which you can download below. Clarity City is an improved version of Metropolis, but has not been updated in Clarity Angular due to the potential for breaking changes.
 
 <div cds-layout="m-t:md">
-<a class="btn btn-primary asset-download-btn" target="_blank" href="https://github.com/vmware/clarity/files/5425574/Metropolis.zip"><cds-icon shape="download"></cds-icon> Clarity City (1.0.0)</a>
+<a class="btn btn-primary asset-download-btn" target="_blank" href="https://github.com/vmware/clarity/files/5425574/Metropolis.zip"><cds-icon shape="download"></cds-icon> Metropolis (1.001)</a>
```
Comment (gnomeontherun) [inline apps/website/get-started/design.md]: Thanks for catching this, but it should probably read `Metropolis M8`, where M8 is the version we originally used and subsequent updates cause layout issues.

Comment (astorije) [reply apps/website/get-started/design.md]: Thanks, I wasn't sure which version to use so I referred to the file metadata, updated!

**Comment by astorije** [comment]
Uuuugh I forgot to add the sign-off thing on that one commit. I did it all from the GitHub UI, can this just go as is? 🙏

**Comment by mathisscott** [comment]
> Uuuugh I forgot to add the sign-off thing on that one commit. I did it all from the GitHub UI, can this just go as is? 🙏

Sorry, @astorije but it can't. This isn't a clarity requirement but a vmware one so we have no wiggle room on its application.

[Amending your commit message](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/changing-a-commit-message) should be pretty straightforward through the command line, however.

**Comment by astorije** [comment]
Ahah yeah I know how to use git. I was hoping to save 2 minutes by using GitHub's great UI editor (e.g. on phone).

Anyway, done.

**Comment by gnomeontherun** [comment]
> Ahah yeah I know how to use git. I was hoping to save 2 minutes by using GitHub's great UI editor (e.g. on phone).

I feel your pain, it ends up being more work to use GitHub's editor. Codespaces might help this, but they also have to be configured to your git account which takes time. Someday we'll have it figured out.

**Comment by astorije** [comment]
I just need to remember it better 😅
Or we could pressure GitHub to add this option to their editor...

**Review by hippee-lee — APPROVED** [review]

**Review by mathisscott — APPROVED** [review]

**Comment by mathisscott** [comment]
@gnomeontherun : Are we going to merge this PR?

**Comment by gnomeontherun** [comment]
It has conflicts, but otherwise we could merge.

**Comment by astorije** [comment]
@gnomeontherun, @mathisscott, rebased on `next` and resolved conflicts.

**Pull request merged**